setup.py

```diff
@@ -61,7 +61,7 @@
     include_package_data=True,
     install_requires=[
         "importlib-metadata ; python_version<'3.8'",
-        "py-solc-x>=1.1.0,<1.2.0",
+        "py-solc-x>==1.1.0",
```

Comment:
```suggestion
        "py-solc-x>=1.1.0",
```

Reply:
I changed it to `==` because I wanted to import a non-public facing helper method. I worry it will change and then the plugin will break.

Reply:
Maybe I should make the change in `solcx`, that would be better.

Right now in solcx, you can't very easily get the commit hashed versions unless you are using the package install. It is small change to be able to support vvm-managed commit hash outputs.

Reply:
https://github.com/iamdefinitelyahuman/py-solc-x/pull/153/files

Reply:
Pinning no longer necessary from 438dbe2